Payment reference

hjblok opened the pull request:
I've added a PaymentReferenceValidator for validating the payment reference (betalingskenmerk). I've used the validation calculations from [Specificaties van de acceptgiro](http://www.acceptgiro.nl/Downloads/Cu_RR_Ac_BijlageG_Formspecs.pdf).

hjblok committed:
Added validation for the Dutch payment reference numbers (betalingskenmerk)

The payment reference number is validated using a weighted modulus 11

hjblok committed:
corrected spelling/markup

sytzeloor commented:
Could you please check out the tests, as they are currently failing with Ruby 1.8.7. See the comment from the Travis Bot above.

hjblok commented:
Yep, I'm working on it ;-)

hjblok committed:
make code compatible with ruby 1.8.7

"a"[0] in 1.8.7 returns an integer representing the character's ASCII
value
"a"[0] in 1.9 returns the actual character

hjblok committed:
cleaning up a little bit
